src/components/newsArchive.js

```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import { graphql, useStaticQuery } from "gatsby"
+import "./newsArchive.scss"
+
+import NewsItem from "./newsItem"
+
+export default function NewsArchive() {
+  const data = useStaticQuery(graphql`
+    {
+      allContentfulNewsItem(
+        sort: { fields: date, order: DESC }
+        filter: { title: { ne: "Why #ArtsMatter" } }
+      ) {
+        edges {
+          node {
+            title
+            summary {
+              summary
+            }
+            contentful_id
+            date
+            headerImage {
+              fixed {
+                src
+              }
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const newsItems = data.allContentfulNewsItem.edges.reduce((news, curNews) => {
```

Comment:
I'm not sure we need to do this. Instead of removing items that don't have a title, date, id, summary, or image, I think we should pass all of the items through and do a conditional within individual news items to display whatever content is entered. Let me know if you want to go over this.